src/js/apps/patients/patients-main_app.js

```diff
@@ -24,11 +24,12 @@ export default RouterApp.extend({
   },
 
   defaultRoute() {
-    const defaultRoute = 'patients:all';
+    const defaultRoute = 'worklist';
+    const defaultWorklist = 'owned-by-me';
 
     this.routeAction(defaultRoute, () => {
-      this.replaceRoute(defaultRoute);
-      this.showPatientsAll();
+      this.replaceRoute(defaultRoute, defaultWorklist);
```

Comment:
This will fix the `'event-router', 'default'` issue:
```js
    this.routeAction(defaultRoute, () => {
      _.defer(()=> {
        this.replaceRoute(defaultRoute, defaultWorklist);
        Radio.request('nav', 'select', this.routerAppName, defaultRoute, [defaultWorklist]);
        this.showPatientsWorklist(defaultWorklist);
      });
    });
```